rollup.config.mjs

```diff
@@ -112,17 +108,14 @@ export default {
       ]
     }),
     alias({
```

Comment:
The idea is great, but the name `makeAlias` suggests we can make it return an entire alias object:
```js
alias: makeAlias({
....

function makeAlias({alias, noop, shim, ...opts}) {
  return { entries, ...opts };
}
```